sdks/python/setup.py

```diff
@@ -219,11 +227,15 @@ def run(self):
     install_requires=REQUIRED_PACKAGES,
     python_requires=python_requires,
     test_suite='nose.collector',
-    tests_require=REQUIRED_TEST_PACKAGES,
+    tests_require=[
+        REQUIRED_TEST_PACKAGES,
+        INTERACTIVE_BEAM,
```

Comment:
Why tests would require interactive beam packages?

Reply:
Because there are unit tests around interactive beam packages.
When those packages are under REQUIRED_PACKAGES, they are always installed and available during tests.
Once we move them into an extras_require, they are not installed nor available during tests anymore.
For example, *-gcp tests will install [gcp,test] to pick up the extras_package GCP_REQUIREMENTS.
Since we don't have/need *-interactive test suites, we can make the group to always be installed as required test packages for tests.

Reply:
We can add a tox suite to test interactivity similar to gcp suites and not install these dependencies for all tests.

Reply:
Yes, I agree! This could avoid running those tests when python version is under 3.5.3.
I guess then the only task would require [interactive] for older versioned python will be "docs" where it scans the source code. I'll just add the try-import statements everywhere to work around it.